### ISSUE TITLE
Panel Intermittently Disappears on Wake

### ISSUE THREAD

**Issue by sclel016** [home-sweet-gnome/dash-to-panel#1884]
- Confirmed the problem persists when `Dash-to-Panel` is the only extension enabled
- Log
```
meta_window_set_stack_position_no_sync: assertion 'window->stack_position >= 0' failed
libinput error: event9  - ELAN0672:00 04F3:3187 Touchpad: kernel bug: Touch jump detected and discarded.
See https://wayland.freedesktop.org/libinput/doc/1.23.0/touchpad-jumping-cursors.html for details
libinput error: event9  - ELAN0672:00 04F3:3187 Touchpad: kernel bug: Touch jump detected and discarded.
See https://wayland.freedesktop.org/libinput/doc/1.23.0/touchpad-jumping-cursors.html for details
libinput error: event9  - ELAN0672:00 04F3:3187 Touchpad: kernel bug: Touch jump detected and discarded.
See https://wayland.freedesktop.org/libinput/doc/1.23.0/touchpad-jumping-cursors.html for details
libinput error: event9  - ELAN0672:00 04F3:3187 Touchpad: WARNING: log rate limit exceeded (5 msgs per 24h). Discarding future messages.
libinput error: event1  - Lid Switch: client bug: event processing lagging behind by 2266ms, your system is too slow
meta_window_set_stack_position_no_sync: assertion 'window->stack_position >= 0' failed
```
- Possibly related to #1236 

**Describe the bug**
After closing and opening laptop lid (triggering sleep/wake and lock/unlock on Fedora 38), the panel disappears when Firefox is focused. I am unable to reproduce the issue with other windows focused. The panel will briefly reappear on mouse over but quickly goes away.  I am unable to capture the problem using OBS even while the issue is occurring on the monitor (very weird). I can take a video with my phone if it would help.

The issue never occurred on Fedora 37.

**Linux distribution and version**
Fedora 38

**GNOME Shell version**
`GNOME Shell 44.0`

**Dash-to-Panel version**
56

**Where was Dash-to-Panel installed from?**
Using `gext` tool from https://github.com/essembeh/gnome-extensions-cli

**Comment by tamasd** [comment]
I have the same issue on a brand new Fedora install (Fedora 38 Workstation, using Wayland).

For me, it happens around 50% of the wakes. Making a previously maximized window not maximized and maximized again makes the problem go away.

My error log:
```
meta_window_set_stack_position_no_sync: assertion 'window->stack_position >= 0' failed
Can't update stage views actor <unnamed>[<Gjs_dash-to-panel_jderose9_github_com_windowPreview_PreviewMenu>:0x5625b6f67830] is on because it needs an allocation.
Can't update stage views actor <preview-menu>[<StWidget>:0x5625b6502220] is on because it needs an allocation.
Can't update stage views actor <dashtopanelPreviewScrollview>[<StScrollView>:0x5625b6553450] is on because it needs an allocation.
Can't update stage views actor <unnamed>[<StBoxLayout>:0x5625b2c2e420] is on because it needs an allocation.
Window manager warning: Invalid WM_TRANSIENT_FOR window 0x2800005 specified for 0x280003f.
Can't update stage views actor <preview-menu>[<StWidget>:0x5625b6502220] is on because it needs an allocation.
Can't update stage views actor <dashtopanelPreviewScrollview>[<StScrollView>:0x5625b6553450] is on because it needs an allocation.
Can't update stage views actor <unnamed>[<StBoxLayout>:0x5625b2c2e420] is on because it needs an allocation.
```

I installed the extension from the gnome extensions website.

**Comment by sclel016** [comment]
Does this happen with windows from applications other than Firefox? I started using Chromium as a mitigation and I have yet to see the issue occur.

**Comment by tamasd** [comment]
Looks like that this only affects Firefox. It seems that if Firefox is in full screen, coming back from sleep it thinks that it should occupy the actual full screen not the full screen minus the panel. My current workaround is to use the intellihide.

**Comment by BensonBear** [comment]
No, it affects EVERY WINDOW not just firefox, and it has been doing this for TWO YEARS

**Comment by devmattrick** [comment]
This is still an issue I'm experiencing.

**Comment by BensonBear** [comment]
> This is still an issue I'm experiencing.

Yes absolutely, all the time.   Also when switching external monitor on and off.   Shocking it would be "marked as stale" when it is like the major issue with the extension.

**Comment by ll2jwork** [comment]
I also experience this issue every time I lock my screen and unlock it. Dash to Panel disappears.

**Comment by BensonBear** [comment]
> I also experience this issue every time I lock my screen and unlock it. Dash to Panel disappears.

What version of gnome and dash-to-panel are you using?  

 I get this problem in gnome 44.10 and dash-to-panel 56 on my older desktop fedora core 38 system.    
 
However, it seems that on my fedora core 39 notebook with gnome 45.5 and dash-to-panel 60, I am finally no longer getting the problem.    
 
I hope you are using an older one since then there is some hope you can fix it with an upgrade, otherwise, we might all be stuck with it still happening even if I currently seem not to be getting it myself on the newer system. 
 
I will be upgrading both machines (and others) to fedora core 40 soon.

**Comment by charlesg99** [comment]
Seems like it is fixed on newer version, gonna close this, thanks!
